### DISCUSSION TITLE
Welcome to Girder Discussions!

### DISCUSSION THREAD

**Discussion by marySalvi** [girder/girder#3575] [Announcements]
## 👋 Welcome!
  This is a forum for Girder users and developers to ask questions and collaborate.

[Girder](http://girder.readthedocs.io/) is a free and [open source](https://github.com/girder/girder) web-based data management platform, developed by [Kitware's](https://www.kitware.com/) Data and Analytics team.

Use this forum to :
  * Ask questions you’re wondering about.
  * Share ideas.
  * Give feedback.
  * Track announcements.
  * Discuss programming and development

**Please note** 
  * Discussions will be elevated to issues when appropriate.
  * If you are looking for help with [Digital Slide Archive](https://github.com/DigitalSlideArchive) or [HistomicsTK](https://github.com/DigitalSlideArchive/HistomicsTK) please use the [DSA Discussion Board](https://github.com/DigitalSlideArchive/digital_slide_archive/discussions).
   Not sure? Please post here and we will be happy to help.

**Comment by pidamakanti** [comment]
Hi, 

We are using opensource Girder data management tool in our GitLab pipelines for upload and download the data. Lately, we are seeing a good number of failures with "HTTP response code said error" for 1MB files to 50MB files. Can someone help to understand why is that so?
